config/database.yml.example

```diff
@@ -12,3 +12,7 @@ development:
 test:
   <<: *default
   database: incidents_test
+
+claims_development:
+  <<: *default
```

Comment:
This also gives us the freedom to specify a different DB adapter, in case @werebus's concern in https://github.com/umts/incidents/pull/132#pullrequestreview-102722231 were to occur.